src/ui/common/src/components/workflows/nodes/MetricOperatorNode.tsx

```diff
@@ -12,13 +23,123 @@ type Props = {
 export const metricOperatorNodeIcon = faTemperatureHalf;
 
 const MetricOperatorNode: React.FC<Props> = ({ data, isConnectable }) => {
+  const defaultLabel = 'Metric';
+  const label = data.label ? data.label : defaultLabel;
+  const currentNode = useSelector(
+    (state: RootState) => state.nodeSelectionReducer.selected
+  );
+  const workflowState = useSelector(
+    (state: RootState) => state.workflowReducer
+  );
+  const selected = currentNode.id === data.nodeId;
+  let execState: ExecState;
+  if (data.nodeType === ReactflowNodeType.Operator) {
```

Comment:
Same as above, we already know here that the node is an operator, so can remove this and just make execState a const.